docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5472,6 +5472,78 @@ the purpose of adding features for the P4Runtime API.
 | `@p4runtime_translation` | See sections [#sec-user-defined-types], [#sec-translation-of-port-numbers] |
 +--------------------------+---------------------------------------+
 ~
+### Custom Annotations in P4Info
+P4 allows flexible annotations. Users may add their own annotations as long as they obey the language syntax.
+To facilitate consumption of user-defined annotations, P4Info has an `anno_map` for any element which can accept
+such annotations, e.g. the `PkgInfo`, `Preamble`, `MatchField`, `ActionRef`, `Action` and
```

Comment:
I suggest replacing "e.g." with "&ie;" as it seems you are giving the exhaustive list of elements for which we have annotations

Reply:
Agreed, will do.